Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_MachException.c

```diff
@@ -321,10 +321,12 @@ handleExceptions(void *const userData)
                 break;
             }
 
-            SentryCrashLOG_DEBUG("Received exception from subprocess. Ignoring message and waiting for another.");
+            SENTRY_ASYNC_SAFE_LOG_DEBUG(
```

Comment:
Assuming this is the only actual change?

Reply:
Yeah I don't know how that other commit got in here.  Lemme see if I can clean that up real quick.

Reply:
Oh interesting looks like Sentry has a bot that automatically adds a commit to PRs that fixes formatting issues?  I'm going to revert that commit and force-push and then merge; don't want to potentially have problems with merge conflicts in the future.